tasks.yaml

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2023-Present The UDS Authors
+
+includes:
+  - test: tasks/tests.yaml
+  - schema: tasks/schema.yaml
+
+variables:
```

Comment:
I dont think you actually need to declare the variables here, since they are set below you wont be able to override them and they're not getting passed anywhere.